cms/db/contest.py

```diff
@@ -193,6 +196,21 @@ class Contest(Base):
         nullable=False,
         default=datetime(2100, 1, 1))
 
+    # Beginning and ending of the contest anaylsis mode.
+    analysis_enabled = Column(
+        Boolean,
+        nullable=False,
+        default=False,
+    )
+    analysis_start = Column(
+        DateTime,
+        nullable=False,
+        default=datetime(2100, 1, 1))
+    analysis_stop = Column(
+        DateTime,
+        nullable=False,
+        default=datetime(2100, 1, 1))
```

Comment:
I like having times - for online contests it could be nice to show "X minutes to start of analysis".